### PR TITLE
updates javascript classes challenges

### PR DESCRIPTION
**Pull request by elysemontano**
updates javascript classes challenges

### CODE REVIEW

File: javascript/classes.md [hunk 1]
```diff
@@ -232,10 +232,8 @@ console.log(bella.description())
 ```
 
 ## Challenges
-
-1. Coffee Maker
-
-```javascript
+``` java
```
Comment (sjproctor) [inline javascript/classes.md:235]: Is this a coffee joke? haha

File: javascript/classes.md [hunk 2]
```diff
@@ -263,27 +261,66 @@ class Coffee {
...
+    this.flavor = flavor
+    this.milkType = milkType
+    this.shots = shots
+  }
 
-- Write a method for your Latte class that outputs the latte's profile.
+  latteProfile() {
+    return `A ${this.flavor} latte with ${this.milkType} milk and ${this.shots} shot(s) of esspresso`
+  }
+}
```
Comment (sjproctor) [inline javascript/classes.md:288]: Small typo in espresso

**Review by sjproctor — CHANGES_REQUESTED** [review]
Beautiful! This section looks great. Just a couple very tiny things.

File: javascript/classes.md (hunk 1 above)
Comment (elysemontano) [reply javascript/classes.md:235]: 🤣  unintentionally yes!

**Review by sjproctor — APPROVED** [review]
Beautiful!

**Pull request merged**